src/utils/Parsing.fsi

```diff
@@ -84,20 +100,29 @@ type internal Tables<'tok> =
 
     /// Interpret the parser table taking input from the given lexer, using the given lex buffer, and the given start state.
     /// Returns an object indicating the final synthesized value for the parse.
-    member Interpret :  lexer:(LexBuffer<char> -> 'tok) * lexbuf:LexBuffer<char> * startState:int -> obj 
+    member Interpret :  lexer:(LexBuffer<'char> -> 'tok) * lexbuf:LexBuffer<'char> * startState:int -> obj 
 
-/// Indicates an accept action has occurred
+#if INTERNALIZED_FSLEXYACC_RUNTIME
```

Comment:
What is the purpose with INTERNALIZED_FSLEXYACC_RUNTIME?
When we would chose one over the other?